### ISSUE TITLE
Can't parse a usfm file that uses .GRK as the file extension.

### ISSUE THREAD

**Issue by davidbaines** [sillsdev/machine#265]
The file extension is given in the project's Settings.XML file and typically includes ".SFM" in the 'PostPart' of the file name.
`<Naming PrePart="" PostPart="usNIV11.SFM" BookNameForm="41MAT" />`

For the Greek resource the USFM files have '.GRK' as their file extension and the line in the Settings.XML is:
`Naming PostPart=".GRK" BookNameForm="41MAT"></Naming>`

Using silnlp.common.check_books to check the GRK project fails to find any SFM files.

**Comment by Enkidu93** [comment]
Does this issue belong in the silnlp repo, @davidbaines?

**Comment by johnml1135** [comment]
I am assuming that this is a USFM parsing issue that needs to be addressed both here (in Machine) and in Machine.py.  Would this be a good one for @mudiagaobrikisil?

**Comment by davidbaines** [comment]
Yes, I thought we're using the Machine / Machine.py USFM parser in silnlp.
I saw the problem in silnlp but thought that it would be in Machine that it
needs to be resolved.

**Comment by ddaspit** [comment]
This seems to be an [issue](https://github.com/sillsdev/silnlp/blob/73e7c5507acb26a0d337b516722d9ed987d939de/silnlp/common/check_books.py#L115) in silnlp. I will transfer the issue to silnlp.
